From 3f833d4214b89d261665d2c1fce48c6688cbeaad Mon Sep 17 00:00:00 2001
From: chandavath <rcrajagopal@habana.ai>
Date: Tue, 22 Oct 2024 15:37:21 +0300
Subject: [PATCH] [FSW-0] fs: run arcmgmt app from arc1 mgmt iccm

This patch enables to run arcmgmt app from arc1 mgmt iccm.

Change-Id: I31388453044f3e0bca24e65152845c726ebac674
Signed-off-by: chandavath <rcrajagopal@habana.ai>
---

diff --git a/boards/intel/fs/fs1_mgmt_palad_defconfig b/boards/intel/fs/fs1_mgmt_palad_defconfig
index 6e58314..6e3d099 100644
--- a/boards/intel/fs/fs1_mgmt_palad_defconfig
+++ b/boards/intel/fs/fs1_mgmt_palad_defconfig
@@ -20,3 +20,5 @@
 CONFIG_INTEL_FS_ICTL=n
 CONFIG_HL_EFUSE=y
 CONFIG_INTEL_GPU_INTR=y
+CONFIG_MGMT_RUN_FROM_ICCM=y
+CONFIG_HARVARD=n
